source/common/router/route_config_update_receiver_impl.cc

```diff
@@ -47,18 +57,24 @@ void RouteConfigUpdateReceiverImpl::initializeVhosts(
   }
 }
 
+// returns true if one or more virtual hosts have been removed from vhosts
 void RouteConfigUpdateReceiverImpl::removeVhosts(
-    std::unordered_map<std::string, envoy::api::v2::route::VirtualHost>& vhosts,
+    std::map<std::string, envoy::api::v2::route::VirtualHost>& vhosts,
     const Protobuf::RepeatedPtrField<std::string>& removed_vhost_names) {
   for (const auto& vhost_name : removed_vhost_names) {
     vhosts.erase(vhost_name);
   }
 }
 
+// returns true if vhosts have been added or updated.
```

Comment:
The comment is out of sync: `return true`?

Reply:
removed the comment